tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -133,84 +132,31 @@ def test_get_metrics_and_expectations(
     )
     assert len(validator.batches) == 36
 
+    # fmt: off
```

Comment:
All changes below are purely cosmetic - we don't need `black` for giant code dumps like this. Removing formatting cuts down LOC by 1000 and improves readability.